lib/migrate/migrate.go

```diff
@@ -678,6 +735,15 @@ func convertLogLevel(logLevel string) string {
 	}
 }
 
+// NilStatusWriter implments the StatusWriterInterface for use by non-cli
```

Comment:
Is this supposed to be part of this PR?

Reply:
I added it because I was updating my calico-node startup PR for these changes and saw the need for that and was thinking Typha could make use of it too. Want me to remove it?